src/me/delta2force/redditbrowser/RedditBrowserPlugin.java

```diff
@@ -184,7 +184,7 @@ public void setRoom(Location l, String submissionId) {
 			spawnHologram(bl.clone().add(.5, .5, .5), title);
 		}
 		
-		spawnHologram(bl.clone().add(.5, -.25, .5), "�6"+s.getScore());
+		spawnHologram(bl.clone().add(.5, -.25, .5), "" + (char)(0xfeff00a7) + "6" + s.getScore());
```

Comment:
Any reason why this isn't using ChatColor api?

Reply:
Would you like to implement it? I created this issue #2 for you.